code/SINGE_GLG_Test.m

```diff
@@ -9,18 +9,23 @@ function SINGE_GLG_Test(Data,varargin)
 params = parseParams(Data,varargin{:})
 mkdir(params.outdir)
 %%% End InputParser
-tic;
 if isdeployed
     set(0,'DefaultFigureVisible','off');
 end
-resampling_method = {'holes';'burst'};
-
 
 % Use a temporary matfile to track intermediate state
 % Create new file for each job ID to avoid bugs when storage is shared
 % between parallel jobs
 m = matfile(['TempMat' '_' num2str(params.ID)],'Writable',true);
```

Comment:
Does support for branching affect how we need to name this temp file to avoid conflicts?  I don't believe it does but wanted to confirm.

Reply:
No, the same temp file can be used.